python/tests/conftest.py

```diff
@@ -200,19 +213,23 @@ def get_online_config(self):
 
             tmpdb = tmpdir.join("livedb%d" % self.live_count)
             ac = self.make_account(tmpdb.strpath, logid="ac{}".format(self.live_count))
-            self._preconfigure_key(ac, configdict['addr'])
+            if pre_generated_key:
+                self._preconfigure_key(ac, configdict['addr'])
```

Comment:
Guess this line fixes `test_ac_setup_message` generating the key. I'll test it and approve after that.